src/styl/now-value.styl

```diff
@@ -0,0 +1,22 @@
+@import '_variables'
+
+.dollert-now-value-container
+  padding-bottom: 15px
+  text-align: center
+  line-height: 20px
+  background $color-white
+  border-bottom 1px solid $color-grey
+  .dollar-now-value-label
+    font-size: 12px
+  .dollar-now-value-field
+    color: $color-primary
+  .dollar-now-value-variation
+    font-size: 12px
```

Comment:
As I can see, `12px` seems to be the default font size inside `dollert-now-value-container`, so move this rule to `dollert-now-value-container` and add a `font-size` into `dollar-now-value-field` with its proper size to avoid repetition.